### PR TITLE
chore: Update dependencies

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

**Comment by jskeet** [comment]
Hmm... will need to work out what's happening on the conformance test front. I may just need to update...

File: examples/Google.Cloud.Functions.Examples.StorageImageAnnotator/Function.cs [hunk 1]
```diff
@@ -41,7 +41,7 @@ public class Startup : FunctionsStartup
     {
         public override void ConfigureServices(WebHostBuilderContext context, IServiceCollection services) =>
             services
-                .AddSingleton(ImageAnnotatorClient.Create())
+                .AddImageAnnotatorClient()
```
Comment (amanda-tarafa) [inline examples/Google.Cloud.Functions.Examples.StorageImageAnnotator/Function.cs:44]: Yay!

**Review by amanda-tarafa — APPROVED** [review]
LGTM

File: examples/Google.Cloud.Functions.Examples.StorageImageAnnotator/Function.cs (hunk 1 above)
Comment (jskeet) [reply examples/Google.Cloud.Functions.Examples.StorageImageAnnotator/Function.cs:44]: Yay for the one we *can* do - shame about the storage one. But I've filed a feature request for that :) I haven't checked build/deployment time, but with any luck not installing Grpc.Core should make this significantly faster.

**Pull request merged**